### ISSUE TITLE
Missing "creator" information/sample

### ISSUE THREAD

**Issue by cyberphone** [w3c-ccg/ld-signatures#6]
I'm unable "deciphering" how the dual-linked "creator" scheme is supposed to work and where exactly the keys are.  It apparently does something quite different to
https://cyberphone.github.io/doc/security/jcs.html#remotekeyexample
but I don't understand what :-(

**Comment by msporny** [comment]
It's not that different to your remotekey example, there is just more data there that enables you to provide access control in a more flexible way.

The dual-linked mechanism is a way of tying a key to an "identity profile". An identity profile effectively expresses an entity (e.g. person, organization, AI, etc.). The key says "this is the identity profile that I'm associated with", and the identity profile says "I am in control of these keys". If the "creator" of a signed message resolved to a key, and they key lists its identity profile and the identity profile of the key lists the key (dual-linked relationship), and there is a digital signature on the identity profile, then you can be sure that the relationship between the key to it's owner is cryptographically valid (there are other important details, but that's the gist of it).

When you can tie a key to an identity profile, you can start making access control decisions on that identity profile without having to track all of their keys. In short, it enables you to effectively say "I trust this entity, let them in", rather than "I trust this key, let anyone with the key in".

In other words, the Linked Data Signatures and Web Key mechanisms enable what JCS does and more.

**Comment by cyberphone** [comment]
> When you can tie a key to an identity profile, you can start making access control decisions on that identity profile without having to track all of their keys. In short, it enables you to effectively say "I trust this entity, let them in", rather than "I trust this key, let anyone with the key in".

This sound like a certificate with one major difference, there's no trust anchor/issuer.

There must be some pretty vital information missing here.

**Comment by dmitrizagidulin** [comment]
The `creator` field is no longer part of the spec, this issue can be closed.
